### PR TITLE
change project name to astro in pom.xml

### PR DESCRIPTION
**Pull request by jackylk**
1. change project name to astro
2. add license information

### CODE REVIEW

**Comment by WangTaoTheTonic** [comment]
I think the groupId should be com.huawei.bigdata or something like that while the artifact id should be astro.

**Comment by ravipesala** [comment]
Can one of the admins verify this patch?

**Comment by ravipesala** [comment]
ok to test

File: pom.xml [hunk 1]
```diff
@@ -26,12 +26,19 @@
     <artifactId>apache</artifactId>
     <version>14</version>
   </parent>
-  <groupId>Huawei-Spark</groupId>
-  <artifactId>spark-sql-on-hbase</artifactId>
+  <groupId>Astro</groupId>
+  <artifactId>astro-parent_2.10</artifactId>
```
Comment (ravipesala) [inline pom.xml]: Jars are created like astro-parent_2.10-1.0.0.jar, can't we just use astro-project  as artifactId

**Comment by ravipesala** [comment]
retest this please

**Comment by ravipesala** [comment]
retest this please

**Comment by ravipesala** [comment]
LGTM. Thanks for fixing it.

**Pull request merged**